api/graphql/searchQuerySet.js

```diff
@@ -1,26 +1,48 @@
 import { mapValues } from 'lodash'
+import { PAGINATION_TOTAL_COLUMN_NAME } from '../../lib/graphql-bookshelf-bridge/util/applyPagination'
 
 export default function searchQuerySet (searchName, options) {
```

Comment:
this splits into `searchQuerySet` and `fetchSearchQuerySet`; the former just prepares a Bookshelf query, and the latter actually runs it. There's definitely some DRYing up that can happen around this, since the adding of `hasMore` to the return value happens both below and in `fetcher.js`.